yarn.lock

```diff
@@ -2966,6 +2966,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/swappable-obj-proxy@npm:^2.2.0":
```

Comment:
Could you run `yarn dedupe` to get rid of the reference to the old version? This is in-range of `^2.1.0` still, which is what's keeping the `2.1.0` around in the lockfile